mongoose/mongoose.d.ts

```diff
@@ -871,7 +871,7 @@ declare module "mongoose" {
     /** Hash containing current validation errors. */
     errors: Object;
     /** The string version of this documents _id. */
-    id: string;
+    id?: string;
```

Comment:
@cyrixmorten woops sorry about that, can you try this one again. I used your idea above.